client/state/data-layer/wpcom/read/site/index.js

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
I think we can ditch the external deps block if there aren't any (linter shouldn't complain).